packages/stylable.io/docs/references/imports.md

```diff
@@ -6,39 +6,95 @@ layout: docs
 
-You use the **Stylable** syntax beginning with `-st-` for the `:import` config:
+## Import syntax
+**Stylable** supports two types of import syntax that can be used, the two are identical in their capabilities.
+- `@st-import` at-rule directive - A newer more concise syntax, similar to ES imports:
+    * replace `<DEFAULT_NAME>` with your desired local name to import an entire stylesheet, representing its root class
+    * Inside the square brackets, import any inner parts belonging to said stylesheet
 
-* `-st-from:` Identifies the path to the stylesheet or JavaScript module. Can be a relative path or a 3rd party path.
-* `-st-default:` Imports the default export of the module named in `-st-from:`. Use with the name by which to identify the imported value in the scoped stylesheet.
-* `-st-named:` List of the named exports to import into the local scoped stylesheet from the file named in `-st-from:`.
+```css
+@st-import <DEFAULT_NAME>, [<NAMED_PART1, NAMED_PART2, ...>] from './stylesheet.st.css';
+```
+
+- `:import` ruleset directive - The original more verbose way of importing symbols, uses the **Stylable** syntax beginning with `-st-` inside the `:import` ruleset:
+    * `-st-from:` Identifies the path to the stylesheet or JavaScript module. Can be a relative path or a 3rd party path.
+    * `-st-default:` Imports the default export of the module named in `-st-from:`. Use with the name by which to identify the imported value in the scoped stylesheet.
+    * `-st-named:` List of the named exports to import into the local scoped stylesheet from the file named in `-st-from:`.
+
+```css
+:import {
+    -st-from: './stylesheet.st.css';
+    -st-default: <DEFAULT_NAME>;
+    -st-named: <NAMED_PART1, NAMED_PART2, ...>;
+}
+```
+
+Every example below will feature both types of import syntaxes, their end result is identical.
 
 > **Note**  
 > * `:import` is a Stylable directive and not a selector.
-> * Using `import` as part of a complex selector or inside a CSS ruleset does not import.
-> * Multiple imports may conflict and the last one in the file wins.
+> * likewise, `@st-import` is a Stylable directive and not an actual at-rule.
+> * Import statements cannot be used as a part of a complex selector or inside a CSS ruleset.
+> * Multiple imports may conflict in their used symbol names; the last one in the file wins.
 
 ## Imports - basic usage
 
 Here are some examples of how you can use imports in your **Stylable** stylesheet.
 
 ### Import the default export of a local reference stylesheet for use in the scoped stylesheet
 
-Import the `toggle-button.css` stylesheet from a local location. Assign the name `ToggleButton` to the default export of that stylesheet for use in this scoped stylesheet. 
+Import the `button.st.css` stylesheet from a local location. Assign the name `Button` to the default export of that stylesheet for use in this scoped stylesheet. 
```

Comment:
suggested rewrite:

Import the `button.st.css` stylesheet from a relative location, and assign the local name `Button` to It's default export root element. 

> **NOTE:**
> Stylesheet default export is always it's root element which is treated as a component and should be named with a capital letter, while the named exports represent other internal parts and definitions (maybe link to "#Import named parts from a local stylesheet")